src/app/features/search/search.component.ts

```diff
@@ -37,7 +37,7 @@ export class SearchComponent {
     return errors.find(error => error.anchorLink === name);
   }
 
-  public get Roles() {
+  public get roles() {
```

Comment:
I think we need to standardise this across the repo, I recently swapped one of these the other way, an annoying job but if you have the time would be great, if not ignore this comment!

Reply:
You already did this lower down lol